ui/src/turing/components/utils/helper.js

```diff
@@ -0,0 +1,11 @@
+export const mapProtocolLabel = (protocol, value) => {
+  if (protocol === "UPI_V1" && value === "payload") {
+    return "prediction context";
+  }
+
+  return value;
+};
+
+export const capitalizeFirstLetter = (string) => {
+  return string.replace(/(^\w|\s\w)/g, (m) => m.toUpperCase());
```

Comment:
Rather than the custom function, we could use this: https://lodash.com/docs#startCase